requirements/how-to-upgrade-python.md

```diff
@@ -47,7 +47,7 @@ Both python and pip version are specified:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python: ["3.9", "3.10"]
+        python: ["3.9", "3.11"]
```

Comment:
I think this is out of date. It should have been `3.10 -> 3.11` and I'd say now it should go to `3.11 -> 3.12`

Reply:
this is just doc, no need to change

you actually need to add the new version in the gh actions config on 

https://github.com/ITISFoundation/osparc-simcore/blob/dc9dc44d168344a9be4294001faf5d6acc27e1b0/.github/workflows/ci-testing-deploy.yml#L1317

i.e. there it should be `["3.11", "3,12"]`